engines/conjur_audit/spec/db_helper.rb

```diff
@@ -2,8 +2,7 @@
 
 shared_context "database setup" do
 
-  let(:db_uri) { ENV['AUDIT_DATABASE_URL'] || 'postgres://postgres@pg/postgres' }
-  let(:db) { Sequel.connect db_uri }
+  let(:db) { Sequel.connect ENV['DATABASE_URL'] }
```

Comment:
is `DATABASE_URL` == 'postgres://postgres@pg/postgres'?

Reply:
yes